### PR TITLE
renaming occurences of standard allowance into performance allowance

### PR DESCRIPTION
**Pull request by ppitou**
renaming standard allowance into performance allowance in core/* editoast/* tests/* python/*

### CODE REVIEW

**Review by flomonster — CHANGES_REQUESTED** [review]
Thanks for your PR!

Some changes required:
- The branch needs to be rebased to fix conflicts.
- We should add a migration to rename the allowance type into `performance` where needed.

File: python/osrd_schemas/osrd_schemas/train_schedule.py
Comment (shenriotpro) [inline python/osrd_schemas/osrd_schemas/train_schedule.py:1]: Please update versions for the 2 Python packages you're updating.

**Review by shenriotpro — APPROVED** [review]
OK for tests/, assuming they pass.

**Comment by flomonster** [comment]
Closing for unactivity and too many conflicts.

**Pull request closed without merging**